NEWS

```diff
@@ -47,6 +47,10 @@ CHANGES WITH 257 in spe:
           migrated between cgroups. It might be fully removed in a future release
           (reach out if you have use cases).
 
+        * The recommended kernel baseline version has been bumped to v5.4.
+          Expect limited testing on older kernel versions. Support for them
+          will be phased out in a future release in 2025.
```

Comment:
Maybe mention the word "minimum baseline" in the 2nd part. maybe append "i.e. the *minimum* baseline will be expected to be bumped to 5.4 then."

Reply:
oh, and maybe mention somewhere that v5.4 is from 2019, i.e. almost 5y old by now.

Reply:
also maybe say that older than v5.4 will now result in a taint string in `systemctl show -P Tainted`